src/manifest.rs

```diff
@@ -72,7 +72,8 @@ pub(crate) fn manifest(uri: &str, resource_type: &str, settings: Option<String>)
     let resource = match resource_type {
         "ClusterAdmissionPolicy" => ClusterAdmissionPolicy::try_from(scaffold_data),
         _other => Err(anyhow!(
-            "Resource {} unknown. Valid types are: ClusterAdmissionPolicy"
+            "Resource {} unknown. Valid types are: ClusterAdmissionPolicy",
+            _other,
```

Comment:
silly question, given we're now using `_other`, shouldn't we change its name to be `other`?

Reply:
Absolutely. Thanks!